# HG changeset patch
# User Benjamin Lindner <lindnerb@users.sourceforge.net>
# Date 1233340756 18000
#      Fri Jan 30 13:39:16 2009 -0500
# Node ID 34b9a57433886c61303842e86b6937590143f01e
# Parent  399af34bb4bf2fdc25c7981baa7f7a2850258e19
fix gzip test code - use filesep()

diff --git a/scripts/miscellaneous/gzip.m b/scripts/miscellaneous/gzip.m
--- a/scripts/miscellaneous/gzip.m
+++ b/scripts/miscellaneous/gzip.m
@@ -48,24 +48,24 @@ endfunction
 %!  unwind_protect
 %!    filename = tmpnam;
 %!    dummy    = 1;
 %!    save(filename, "dummy");
 %!    dirname  = tmpnam;
 %!    mkdir(dirname);
 %!    entry = gzip(filename, dirname);
 %!    [path, basename, extension] = fileparts(filename);
-%!    if ! strcmp(entry, [dirname, "/", basename, extension, ".gz"])
+%!    if ! strcmp(entry, [dirname, filesep, basename, extension, ".gz"])
 %!      error("gzipped file does not match expected name!");
 %!    endif
 %!    if ! exist(entry, "file")
 %!      error("gzipped file cannot be found!");
 %!    endif 
 %!    gunzip(entry);
-%!    if (system(sprintf("diff %s %s/%s%s", filename, dirname, 
+%!    if (system(sprintf("diff %s %s%c%s%s", filename, dirname, filesep,
 %!                                          basename, extension)))
 %!      error("unzipped file not equal to original file!");
 %!    end
 %!  unwind_protect_cleanup
 %!    delete(filename);
-%!    delete([dirname, "/", basename, extension]);
+%!    delete([dirname, filesep, basename, extension]);
 %!    rmdir(dirname);
 %!  end_unwind_protect
